modules/siemens/scanner.go

```diff
@@ -18,6 +19,7 @@ import (
 type Flags struct {
 	zgrab2.BaseFlags `group:"Basic Options"` // TODO: configurable TSAP source / destination, etc
 	Verbose          bool                    `long:"verbose" description:"More verbose logging, include debug fields in the scan results"`
+	ReadTimeout      time.Duration           `long:"read-timeout" default:"10ms" description:"Timeout for reading S7 responses"`
```

Comment:
```suggestion
	ReadTimeout      time.Duration           `long:"read-timeout" default:"500ms" description:"Timeout for reading S7 responses"`
```
Our pattern is usually to give the user sane defaults with possible customization. I think a default value of 500 ms feels better for internet-wide scanning.